src/WireMock.Net/Owin/WireMockMiddleware.cs

```diff
@@ -99,7 +100,7 @@ private async Task InvokeInternal(IContext ctx)
                     return;
                 }
 
-                logRequest = !targetMapping.IsAdminInterface;
+                logRequest = !targetMapping.IsAdminInterface || targetMapping.Provider is ProxyAsyncResponseProvider;

```

Comment:
The mapping class already has this code:
``` c#
public bool IsAdminInterface => Provider is DynamicResponseProvider || Provider is DynamicAsyncResponseProvider || Provider is ProxyAsyncResponseProvider;
```
So no need to add this here?

Reply:
That's the root cause of the problem (there is negation). In this check it is actually assumes that proxy response is NOT admin interface in order to log it. I have tried to remove that condition on mapping but it actually conflicts with Admin interface. There is a test for that - FluentMockServer_Proxy_Should_Not_overrule_AdminMappings. So it is kind of workaround to still log proxy requests.

Reply:
Ah yes, Sorry.

In that case. Maybe it's better to to add a property to the mapping class, like this:

``` c#
public bool LogRequest => !(Provider is DynamicResponseProvider || Provider is DynamicAsyncResponseProvider).
```

Reply:
What about smth like:
`
public bool IsProxyRequest => Provider is ProxyAsyncResponseProvider
`

Reply:
But I then you still need to check if it's not an admin request.

Reply:
In the MiddleWare - yes. Let me update the PR